dashboard/src/components/IasqlEditor.tsx

```diff
@@ -146,9 +146,10 @@ export default function IasqlEditor() {
             .map(tbl => Object.entries(tbl).map(([value, meta]) => ({ value, meta })))
             .flat(),
           // Function Names
-          ...(Array.from(functions) ?? [])
-            .filter((h: any) => !!h?.signature)
-            .map((h: any) => ({ value: h.signature, meta: 'function' })),
+          ...Object.values(functions)
+            .map(mod => Object.keys(mod))
```

Comment:
```suggestion
            .map(fn => Object.keys(fn))
```
The variable itself is named `function`. Therefore the values should be functions?